### DISCUSSION TITLE
make my own back up with all files and models downloaded so i can just install next time i need to reset pc

### DISCUSSION THREAD

**Discussion by Mariehamn360** [lllyasviel/Fooocus#2650] [Q&A] [answered]
Hi! i just want to know if there are ways to for example make an installer for windows that installs fooocus with all models, checkpoints and settings? i often factory reset my pc and would like to have a way to just install and continue where i left instead of having to download all models by my own and do redownload again. is it possible to make an installer and if so how shall i do?

**Comment by ILikeToasters** [comment, marked as answer]
Everything is in that folder by default so you can just archive that folder and move it.
